client/hotrod-client-legacy/src/test/java/org/infinispan/client/hotrod/event/ContinuousQueryObjectStorageTest.java

```diff
@@ -72,7 +73,7 @@ protected void createCacheManagers() throws Throwable {
 
    @Override
    protected List<SerializationContextInitializer> contextInitializers() {
-      return Arrays.asList(TestDomainSCI.INSTANCE, ClientEventSCI.INSTANCE);
+      return Arrays.asList(GlobalContextInitializer.INSTANCE, TestDomainSCI.INSTANCE, ClientEventSCI.INSTANCE);
```

Comment:
I've noticed `GlobalContextInitializer.INSTANCE` is added in most of the tests. Would it make sense if the cache manager automatically adds it?